src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/packageScanner.hpp

```diff
@@ -675,31 +641,14 @@ class TPackageScanner final : public AbstractHandler<std::shared_ptr<TScanContex
 
         try
         {
-            if (!scanPackageTranslation(CNAValue, data, vulnerabilityScan))
-            {
-                const std::string packageName {Utils::toLowerCase(std::string(data->packageName()))};
-                const std::string packageVendor {Utils::toLowerCase(std::string(data->packageVendor()))};
-
-                logDebug1(WM_VULNSCAN_LOGTAG,
-                          "Initiating a vulnerability scan for package '%s' (%s) (%s) with CVE Numbering "
-                          "Authorities (CNA) "
-                          "'%s' on Agent "
-                          "'%s' (ID: '%s', Version: '%s').",
-                          packageName.c_str(),
-                          data->packageFormat().data(),
-                          packageVendor.c_str(),
-                          CNAValue.c_str(),
-                          data->agentName().data(),
-                          data->agentId().data(),
-                          data->agentVersion().data());
-
-                PackageData package = {.name = packageName,
-                                       .vendor = packageVendor,
-                                       .format = data->packageFormat().data(),
-                                       .version = data->packageVersion().data()};
-
-                m_databaseFeedManager->getVulnerabilitiesCandidates(CNAValue, package, vulnerabilityScan);
-            }
+            const std::string packageName {std::string(data->packageName())};
+            const std::string packageVendor {std::string(data->packageVendor())};
+
+            PackageData package = {.name = packageName,
+                                   .vendor = packageVendor,
+                                   .format = data->packageFormat().data(),
+                                   .version = data->packageVersion().data()};
```

Comment:
Prevent unnecessary copies:
```suggestion
            PackageData package = {.name = data->packageName().data(),
                                   .vendor = data->packageVendor().data(),
                                   .format = data->packageFormat().data(),
                                   .version = data->packageVersion().data()};
```